lua/telescope/make_entry.lua

```diff
@@ -445,9 +471,16 @@ function make_entry.gen_from_buffer(opts)
       display_bufname = entry.filename
     end
 
+    local icon, hl_group = get_devicons(entry.filename, disable_devicons)
+
+    if hl_group then
+      icon = { icon, hl_group }
+    end
```

Comment:
You don't need to do that nil check here. Just pass in the table to the `displayer`. If hl_group is nil then we will not do anything :) I think

Reply:
Thank you! I fixed it.